apps/ledger-live-mobile/src/screens/Platform/v2/hooks.ts

```diff
@@ -28,7 +28,8 @@ export function useCatalog() {
   const recentlyUsed = useRecentlyUsed(categories.manifests.all, db);
 
   const search = useSearch<AppManifest, TextInput>({
-    list: categories.searchable,
+    listInput: categories.searchable,
+    listFilter: categories.searchable,
```

Comment:
On LLD we use `categories.manifests.complete` and not `categories.searchable`
The diff between LLD and LLM here is a bit strange